src/dev/mod.ts

```diff
@@ -28,70 +28,31 @@ export function ensureMinDenoVersion() {
   }
 }
 
+async function collectDir(dir: string): Promise<string[]> {
+  const paths = [];
+  const routesFolder = walk(dir, {
+    includeDirs: false,
+    includeFiles: true,
+    exts: ["tsx", "jsx", "ts", "js"],
+  });
+  for await (const entry of routesFolder) {
+    const path = SEP + relative(dir, entry.path);
```

Comment:
I'm a passing Windows machine user. If you change it as follows, it should work on Windows!

```suggestion
async function collectDir(dir: string): Promise<string[]> {
  const dirUrl = toFileUrl(dir);
  const paths = [];
  const routesFolder = walk(dir, {
    includeDirs: false,
    includeFiles: true,
    exts: ["tsx", "jsx", "ts", "js"],
  });
  for await (const entry of routesFolder) {
    const path = toFileUrl(entry.path).href.substring(
      dirUrl.href.length,
    );
```

(need to add `toFileUrl` to the import statement)